.storybook/preview.jsx

```diff
@@ -12,7 +12,7 @@ export const parameters = {
 
 const withChakra = (StoryFn, _context) => {
   return (
-    <ChakraProvider theme={theme} >
+    <ChakraProvider theme={theme('day')} >
```

Comment:
Nice, eventually we can make a control in storybook to toggle the color mode. That can be a separate ticket though.

Reply:
Ticket Created.